.github/workflows/label-commenter.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Automatically respond to any issues/pull requests that have the given labels assigned.
+#
+name: Label Commenter
+
+on:
+  issues:
+    types:
+      - labeled
+      - unlabeled
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  comment:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Label Commenter
+        # Note: Using SHA explicitly for v1.2.3 - https://julienrenaux.fr/2019/12/20/github-actions-security-risk/
+        uses: peaceiris/actions-label-commenter@93941f8f189a4b92ab75059aa39fe421469253f4
```

Comment:
Calling out my previous comment:

> This is an action from the [marketplace](https://github.com/marketplace/actions/label-commenter), I need to confirm if the actions are also security reviewed by Github, or if it's only apps that just reviewed:
> 
> https://developer.github.com/marketplace/getting-started/requirements-for-listing-an-app-on-github-marketplace/#security

I don't think the community actions are reviewed by Github:

> Label Commenter is not certified by GitHub. It is provided by a third-party and is governed by separate terms of service, privacy policy, and support documentation.

To be safe, I'm going ahead an locking this down to a specific SHA that i've reviewed the code for, instead of the previous tag number:

```diff
- uses: peaceiris/actions-label-commenter@v1
+ uses: peaceiris/actions-label-commenter@93941f8f189a4b92ab75059aa39fe421469253f4
```

Based on the article https://julienrenaux.fr/2019/12/20/github-actions-security-risk/

Reply:
Confirming that Github actions aren't reviewed, anyone can publish to Github actions via their Github repo directly:

https://help.github.com/en/actions/building-actions/publishing-actions-in-github-marketplace#publishing-an-action